modules/gdscript/tests/scripts/parser/features/export_variable.gd

```diff
@@ -20,6 +22,10 @@ const Utils = preload("../../utils.notest.gd")
 @export var test_image: Image
 @export var test_timer: Timer
 
+# Custom Scripts
```

Comment:
```suggestion
# Global custom classes.
```

Reply:
Thanks! I've put those changes in the latest commit :)